src/strings/PowerFxResources.en-US.resx

```diff
@@ -1798,6 +1798,10 @@
     <value>Numeric value is too large.</value>
     <comment>Error Message.</comment>
   </data>
+  <data name="ErrTextFormatTooLarge" xml:space="preserve">
+    <value>Format string size is too large.</value>
```

Comment:
><value>Format string size is too large.</value> [](http://example.com/codeflow?start=4&length=47)

nit - can we parameterize this

`Format string size can't be more than {0}  characters`